src/test/JerryProtocolHandler.test.ts

```diff
@@ -15,6 +15,13 @@ function encodeArray(byte: number, str: string) {
   return array;
 }
 
+function addVersion(configArray) {
+  let version = new Uint8Array(4);
```

Comment:
The `addVersion` name should be modified to `createConfiguration`. Please add a comment what this function does exactly. It would be nice, it there could be a comment about the parameter as well:

```javascript
// configArray data: [msgtype, endianness, maxMsgSize, cpointerSize, ...]
```